DigitalLearningSolutions.Data.Tests/Services/DiagnosticAssessmentServiceTests.cs

```diff
@@ -192,61 +180,166 @@ public void Get_diagnostic_assessment_should_return_null_if_diag_status_and_stat
             result.Should().BeNull();
         }
 
-        [TestCase(70093, 3452, 110, 38227)]
-        [TestCase(84238, 6062, 105, 55672)]
-        [TestCase(83361, 5263, 161, 98818)]
-        [TestCase(173505, 11347, 178, 156377)]
-        [TestCase(121301, 5903, 169, 180362)]
-        [TestCase(181938, 5263, 153, 198839)]
-        [TestCase(269044, 10059, 212, 237170)]
-        public void Get_diagnostic_assessment_should_have_same_tutorials_as_stored_procedure(
-            int candidateId,
-            int customisationId,
-            int sectionId,
-            int progressId
-        )
+        [Test]
+        public void Get_diagnostic_assessment_should_not_have_tutorials_with_false_diagStatus()
+        {
+            // Given
+            const int customisationId = 14579;
+            const int candidateId = 102375;
+            const int sectionId = 135;
+
+            // When
+            var result = diagnosticAssessmentService.GetDiagnosticAssessment(customisationId, candidateId, sectionId);
+
+            // Then
+            result.Should().NotBeNull();
+            var tutorialIds = result!.Tutorials.Select(tutorial => tutorial.Id).ToList();
+            tutorialIds.Should().NotContain(541); // Tutorial with DiagStatus 0
+            tutorialIds.Should().Equal(535, 536, 537, 538, 539, 540, 542);
+        }
+
+        [Test]
+        public void Get_diagnostic_assessment_should_not_have_archived_tutorials()
```

Comment:
Could you somehow mark that this has been done for this page on HEEDLS-288, e.g. by creating a subtask for it and moving it to test or adding a comment? 🙂

Reply:
Done! 🙂